docs/typeslots.rst

```diff
@@ -122,7 +122,7 @@ reference cycles:
 
        // If w->value has an associated CPython object, return it.
        // If not, value.ptr() will equal NULL, which is also fine.
-       nb::object value = nb::find(w->value);
+       nb::handle value = nb::find(w->value);
```

Comment:
Does assigning to `handle` actually suffice here? AFAICT, `nb::find()` still returns `object`, so you're still doing an incref/decref pair. Maybe that's OK as long as it's not held across VISIT?

Reply:
The problem is having an artificially increased ref count when calling `Py_VISIT`. We temporarily acquire a reference here but give it back before the critical step.